includes/constants.js

```diff
@@ -0,0 +1,18 @@
+// Note: salesforce schema table names shouldn't be constants
```

Comment:
A bit confused by this file, doesn't lo9ok like it's actually used anywhere though? Maybe just needs deleting :D